pkg/sentry/certs/store.go

```diff
@@ -12,10 +12,7 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/config"
 	"github.com/dapr/dapr/pkg/sentry/consts"
 	"github.com/dapr/dapr/pkg/sentry/kubernetes"
-)
-
-const (
-	defaultSecretNamespace = "default"
+	"github.com/dapr/dapr/utils"
```

Comment:
this package is already used in `sentry/config/config.go`. It's not newly added to this binary. 
https://github.com/dapr/dapr/blob/master/pkg/sentry/config/config.go#L15